### DISCUSSION TITLE
Server returns blank page

### DISCUSSION THREAD

**Discussion by Lorenzo0111** [shlinkio/discussions#56] [Help wanted] [answered]
#### How Shlink is set-up

* Shlink Version: 2.10.2
* PHP Version: 8.1.1
* How do you serve Shlink: Self-hosted nginx
* Database engine used: SQLite

#### Summary
When I try to reach the page it returns a blank page. It's not a nginx problem because I tried setting it to an index.php file that shows an h1 and it works. Also the app.shlink.io says that cannot reach the server.

#### Current behavior
Blank page (https://to.lorenzo0111.me)

#### Expected behavior
Everything works

#### How to reproduce
Visit the page or try to add it inside the shlink panel

**Comment by acelaya** [comment]
Can you check Shlink logs? They are located inside data/log folder.

Nginx logs might also have something useful.

**Comment by Lorenzo0111** [comment]
there isn't any shlink log.
where can i find nginx logs?

**Reply by acelaya** [reply]
It depends on your operating system. I would try googling for "nginx logs default location <os_name>"

**Comment by Lorenzo0111** [comment]
also I can see that the page returns 500

**Reply by acelaya** [reply]
Yep, that's why I suggested checking nginx logs. The reason will be logged there.

My bet, either you are running an unsupported PHP version, you are missing some of the required PHP extensions, or Shlink does not have permissions to write inside some of the data folders.

**Comment by Lorenzo0111** [comment, marked as answer]
Fixed, i wrongly forgot to give write permission to the folder
